### PR TITLE
KAZOO-5362: Exclude assigned devices from ringing when a user is in a queue

### PR DESCRIPTION
**Pull request by harenson**
Required for https://github.com/2600hz/kazoo-qubicle/pull/40 to work

### CODE REVIEW

File: core/kazoo_endpoint/src/kz_endpoint.erl [hunk 1]
```diff
@@ -1131,6 +1131,7 @@ create_sip_endpoint(Endpoint, Properties, #clid{}=Clid, Call) ->
                       ,{<<"Failover">>, maybe_build_failover(Endpoint, Clid, Call)}
                       ,{<<"Metaflows">>, kz_json:get_json_value(<<"metaflows">>, Endpoint)}
                       ,{<<"Endpoint-Actions">>, endpoint_actions(Endpoint, Call)}
+                      ,{<<"Exclude-From-Queues">>, kz_json:get_value(<<"exclude_from_queues">>, Endpoint)}
```
Comment (k-anderson) [inline core/kazoo_endpoint/src/kz_endpoint.erl]: This should not be a new root level property but should be in the existing object:

"qubicle": {
    "enabled": true,
   "presence_id": "7900@test.com",
    "exclude": true
}

**Comment by k-anderson** [comment]
Also, please make sure the schemas are updated :)

**Comment by harenson** [comment]
I am missing to do some changes to this PR, so for the moment this PR is not ready to be merged.

**Comment by mark2600** [comment]
I agree with making it a generic flag

**Pull request closed without merging**